doc/api/stream.md

```diff
@@ -860,6 +860,7 @@ added: v0.9.4
 * `destination` {stream.Writable} The destination for writing data
 * `options` {Object} Pipe options
   * `end` {boolean} End the writer when the reader ends. Defaults to `true`.
+* Returns {stream.Writable} making it possible to set up chains of piped streams
```

Comment:
A nit: this is usually `* Returns:` (with a colon). The same for some other additions.

Reply:
fixed in https://github.com/nodejs/node/pull/19361/commits/51f9814bd745eb4a00080dfd4ca92ed96781920e